src/Faker/Provider/DateTime.php

```diff
@@ -113,11 +115,10 @@ public static function dateTimeBetween($startDate = '-30 years', $endDate = 'now
         }
 
         $timestamp = mt_rand($startTimestamp, $endTimestamp);
-
         $ts = new \DateTime('@' . $timestamp);
         $ts->setTimezone(new \DateTimeZone(date_default_timezone_get()));
 
-        return $ts;
+        return new \DateTime($ts->format($format));
```

Comment:
The purpose of format is the posibility to create a DateTime object based on a format. For example:

<dateTimeBetween('-4 days', 'now', 'Y-m-d')>
This will give me a date between 4 days ago and now but, always at 00:00:00 becase i dont especify the hour.

<dateTimeBetween('-4 days', 'now', 'Y-m-d 8:00')>
This will give me a date between 4 days ago and now but, always at 08:00:00


<dateTimeBetween('-4 days', 'now', 'Y-m-d H:00')>
This will give me a date between 4 days ago and now but, always at actual hour in 0 minutes and 0 seconds

<dateTimeBetween('-4 days', 'now')>
This will give me a date between 4 days ago and now but, always at random time